config/libraries.yml

```diff
@@ -40,7 +40,7 @@ keyano:
   neosurl: "https://www.neoslibraries.ca/member-libraries/keyano-college-library/"
   proxy: "http://ezproxy.afmk.talonline.ca/login?url="
 kings:
-  name: "The King's University"
+  name: "King's University"
```

Comment:
This value has flip/flopped from [fixed](https://github.com/ualbertalib/NEOSDiscovery/issues/129) to [fixed](https://github.com/ualbertalib/NEOSDiscovery/pull/208). It's not obvious until you look at the code that these should be different.

Reply:
![Screenshot from 2019-07-18 14-29-07](https://user-images.githubusercontent.com/1220762/61489667-710ed080-a968-11e9-8883-497e811fa84e.png)
![Screenshot from 2019-07-18 14-28-21](https://user-images.githubusercontent.com/1220762/61489668-710ed080-a968-11e9-95a2-bd09b9ce92e6.png)